tsconfig.json

```diff
@@ -11,7 +11,8 @@
       "@assets/*": ["src/assets/*"],
       "@lib/*": ["src/lib/*"],
       "@api/*": ["src/api/*"],
-      "@sanity/*": ["src/sanity/*"]
+      "@sanity/*": ["src/sanity/*"],
+      "@src/*": ["src/*"]
```

Comment:
I'm guessing this fixes the error I saaw during standup (cannot find module .../const). Good find!